doc/basics

```diff
@@ -167,3 +167,14 @@ launched it using the CLI, the table of results will show the nested tests inden
 
 The `Runner` introduced by the `suite` method is the same as any other `Runner`, so further test suites can be
 defined inside other test suites, making it possible to organise tests into a hierarchy.
+
+### Using sbt
+
+Add to your `build.sbt`:
+
+```
+libraryDependencies ++= Seq(
+  "com.propensive" %% "probably-sbt" % "0.8.0" % "test"
+)
+testFrameworks := Seq(new TestFramework("probably.sbt.Framework"))
```

Comment:
Do you think we could put `Framework` in the `probably` package? I try to avoid having too many packages in libraries, even if they are different modules...